packages/middleware-sdk-s3/src/validate-bucket-name.ts

```diff
@@ -7,16 +7,19 @@ import {
   MetadataBearer,
   Pluggable,
 } from "@aws-sdk/types";
+import { validate } from "@aws-sdk/util-arn-parser";
```

Comment:
import as validateArn?
```suggestion
import { validate as validateArn } from "@aws-sdk/util-arn-parser";
```